04_sessions.html

```diff
@@ -96,7 +96,7 @@
  requires passing a <a>session ID</a>.
 
   <section>
-    <h3>newSession()</h3>
+    <h3>newSession</h3>
```

Comment:
New Session because it's a header and we don't use a command list any longer.  This will match the reference in the HTTP mapping table above.